content/dart/concepts/type-conversion/type-conversion.md

```diff
@@ -0,0 +1,496 @@
+---
+Title: 'Type Conversion'
+Description: 'The process of changing the data type of a value from one type to another.'
+Subjects:
+  - 'Computer Science'
+  - 'Mobile Development'
+  - 'Code Foundations'
+Tags:
+  - 'Dart'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-dart'
+  - 'paths/computer-science'
+---
+
+In Dart, **type conversion** refers to the process of converting a value from one data type to another. Dart supports both Implicit and Explicit type conversion.
+
+## Implicit Type Conversion
+
+Implicit type conversion is done automatically, when operations involving different numeric types are performed.
+
+## Implicit conversion Example 1
+
+```dart
+void main() {
+  int intValue = 10;
+  double doubleValue = .5;
+
+  // Implicit type conversion in arithmetic operation
+  double result = intValue + doubleValue;
+
+  print(result);
+}
+```
+
+The `Shell` output:
+
+```shell
+10.5
+```
+
+In Example 1, the `int` intValue (10) is implicitly converted to a `double` (10.0) to perform the addition with doubleValue.
+
+## Implicit conversion Example 2
+
+```dart
+void main() {
+  int intValue = 42;
+  String stringValue = "The value is: $intValue"; // Implicit toString() call
+  print(stringValue);
+}
+```
+
+The `Shell` output:
+
+```shell
+The value is: 42
+```
+
+The `toString()` method is implicitly called when an object is used in a string interpolation. In Example 2, Dart automatically calls the `toString()` method on `intValue` to obtain the string representation.
+
+---
+
+## Explicit Type Conversion
+
+Explicit type conversion is done by the programmer using predefined functions or methods to convert values from one type to another. The following are the methods for Explicit type conversion:
+
+* `toString()`[^1] : Converts a value to a string.
+* `toDouble()`[^2] : Converts a value to a double.
+* `toInt()`[^3] : Converts a value to an integer.
+* `parse()`[^4] : Converts a string to a numeric type.
+* `tryParse()`[^5]: Similar to `parse()`, but returns `null` if the conversion fails.
+
```

Comment:
Please add one example of an explicit function. Follow the ### Example block for the same.